js/portfolio.js

```diff
@@ -41,10 +31,26 @@ tabContent.forEach(function(a){
   $('#projects').append(a.toHtml())
 });
 
+schoolData.sort(function(a,b) {
+  return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
+});
+
+schoolData.forEach(function(ele) {
+  tabContent.push(new Portfolio(ele));
+})
+
 tabContent.forEach(function(a){
   $('#school').append(a.toHtml())
 });
 
+workData.sort(function(a,b) {
+  return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
```

Comment:
I get an error on this line because the publishedOn field is a string that can't be cast to a date. You should either take the sorting out, or add an actual date to your workData objects.